src/components/Custom/RJSFFormFieldAdapter/CustomPriorityBoundsField.jsx

```diff
@@ -189,13 +320,91 @@ const CustomPriorityBoundsField = (props) => {
           <FormattedMessage {...(isMapVisible ? messages.hideMap : messages.showMap)} />
         </button>
 
+        {/* Upload GeoJSON Button */}
+        <div
+          className={`mr-button mr-button--small mr-flex mr-items-center mr-gap-2 mr-transition-all mr-duration-300 mr-mt-2 mr-py-2 mr-cursor-pointer ${
+            isDragOver
+              ? "mr-button--green mr-bg-opacity-80"
+              : "mr-button--white hover:mr-bg-gray-50"
+          }`}
+          onDragOver={handleDragOver}
+          onDragLeave={handleDragLeave}
+          onDrop={handleDrop}
+        >
+          <input
+            type="file"
+            accept=".json,.geojson"
+            onChange={handleFileInputChange}
+            className="mr-absolute mr-inset-0 mr-w-full mr-h-full mr-opacity-0 mr-cursor-pointer"
+            id={`geojson-upload-${props.name}`}
+          />
+          <SvgSymbol sym="upload-icon" viewBox="0 0 20 20" className="mr-w-5 mr-h-5" />
+          <FormattedMessage {...messages.uploadGeoJSON} />
+        </div>
+
+        {/* Info Icon */}
+        <div className="mr-relative tooltip-container">
+          <button
+            type="button"
+            onClick={(e) => {
+              e.stopPropagation();
+              setShowTooltip(!showTooltip);
+            }}
+            className="mr-ml-1 mr-mt-2 mr-p-1 mr-rounded-full mr-text-gray-500 hover:mr-text-gray-700 hover:mr-bg-gray-100 mr-transition-colors"
+            title="Show GeoJSON format info"
+          >
+            <SvgSymbol sym="info-icon" viewBox="0 0 20 20" className="mr-w-4 mr-h-4" />
+          </button>
+
+          {/* Custom Tooltip */}
+          {showTooltip && (
+            <div
+              style={{ width: "300px" }}
+              className="mr-absolute mr-z-50 mr-text-white mr-text-sm"
+            >
+              <pre className="mr-text-gray-300">
+                <FormattedMessage {...messages.geoJSONFormatInfo} />:
+                {`\n{"features":[{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[lng,lat],[lng,lat],...]]}}]}`}
```

Comment:
I think this tooltip is confusing, because the example object isn't valid GeoJSON (it lacks a `type` field). Instead you should just make the tooltip say "Expects a GeoJSON Feature or FeatureCollection containing Polygon geometry(s)."; the GeoJSON spec ([RFC 7946](https://datatracker.ietf.org/doc/html/rfc7946)) spells out the details.